src/components/All-issues/AllIssues.tsx

```diff
@@ -0,0 +1,238 @@
+"use-client"
+import style from "./index.module.css"
+import style2 from "../main-content/general.module.css"
+import { useTheme } from "../contextApi/ThemeContext"
+import Image from "next/image"
+import * as Dialog from "@radix-ui/react-dialog"
+import { DiscIcon } from "@radix-ui/react-icons"
+import { useState } from "react"
+
+interface Labels2Element {
```

Comment:
lets give this a more descriptive name. I don't know what label2Element is.

Reply:
Alright